README.md

```diff
@@ -0,0 +1,41 @@
+# COVID-19 NZ Data Processing
+
+Script to automatically fetch, process, and plot the NZ covid-19 data from the Ministry of Health's website (https://health.govt.nz).
+
+Getting started:
+- 
+Dependencies:
+- pandas
+- requests
+- xlrd
+- beautifulsoup4
+- matplotlib
+- redis
+
+Simply run:
+
+`pip install git+https://github.com/ABI-Covid-19/moh-data.git`
+
+Example of usage:
+
+This new version of moh-data uses [Redis](https://redis.io/) as an in-memory data solution and message broker.
+You will need to [download](https://redis.io/download) and install redis server in order to be able to run this code. 
+
+Simply, create a `.py` file and copy-paste the code below:
+
+```python
+from moh_data.main import Basic
+
+run_data = Basic()
+run_data.plot_daily_trend()
+run_data.plot_cumulative_sum()
+run_data.plot_daily_arrival_sum()
+```
```

Comment:
I have just tried this, but to no avail. I have called my Python script `data.py` and upon executing it using `python data.py`, I am getting:

```
Traceback (most recent call last):
  File "data.py", line 1, in <module>
    from moh_data.main import Basic
  File "/Users/Alan/virtualenv/moh-data/lib/python3.7/site-packages/moh_data/main.py", line 1, in <module>
    from src.moh_data.core.collector import DataCollector
ModuleNotFoundError: No module named 'src'
```

From there, I edited `/Users/Alan/virtualenv/moh-data/lib/python3.7/site-packages/moh_data/main.py` and removed `src.` from `src.moh_data.core.collector` and `src.moh_data.utils.visualisation`, but still to no avail since I am now getting:

```
Traceback (most recent call last):
  File "data.py", line 1, in <module>
    from moh_data.main import Basic
  File "/Users/Alan/virtualenv/moh-data/lib/python3.7/site-packages/moh_data/main.py", line 2, in <module>
    from moh_data.utils.visualisation import Visualisation
ModuleNotFoundError: No module named 'moh_data.utils'
```

Reply:
And indeed...

```
ll /Users/Alan/virtualenv/moh-data/lib/python3.7/site-packages/moh_data/
total 8
-rw-r--r--  1 Alan  staff     0B  8 Apr 17:36 __init__.py
drwxr-xr-x  4 Alan  staff   128B  8 Apr 17:38 __pycache__
drwxr-xr-x  6 Alan  staff   192B  8 Apr 17:36 core
-rw-r--r--@ 1 Alan  staff   1.4K  8 Apr 17:42 main.py
```

So, no, `utils` folder with a `visualisation.py` file...?